src/DrHuExampleDesignPatterns/AdaptorPattern.java

```diff
@@ -0,0 +1,39 @@
+package DrHuExampleDesignPatterns;
```

Comment:
Instead of storing this here, make a separate folder: `extras` and store this there. `src` should only contain our assignment as I have it laid out on master